cmd/ddev/cmd/describe.go

```diff
@@ -133,4 +258,23 @@ func renderAppDescribe(desc map[string]interface{}) (string, error) {
 
 func init() {
 	RootCmd.AddCommand(DescribeCommand)
+	DescribeCommand.Flags().StringVarP(&service, "service", "s", "", "The service for additional information")
+	DescribeCommand.Flags().BoolVarP(&verbose, "verbose", "v", false, "Show extended output for all services")
+}
+
+func formatStatus(status string) string {
```

Comment:
I think gchalk would be easier than this and far more powerful.

Reply:
Can you give me a quick summary of how to manage dependencies for DDev? I tried updating `gosuri/uitable` using `go get -d github.com/gosuri/uitable@v0.0.4`, which appeared to work, but them `make` started failing because of explicit dependencies in `vendor/modules.txt`. I found references to that with regard to `go mod vendor`, but running that failed due to `embed: package embed is not in GOROOT (/usr/local/go/src/embed)` and I didn't want to go further down the rabbit hole!

Reply:
What I do is the `go get` and then `go mod vendor && go mod tidy`

Reply:
Updating go seemed to fix my issue! So got uitable updated now.

What's the reason for the gchalk switch? Whilst it looks more powerful, nothing I've done so far (or am expecting to do) adds to what was already being done using `fatih/color`, so swapping seems a bit of tangent?

Reply:
I was just thinking you'd have a lot of fun with it :) Might give more flexibility than just colors. It's been on my list to try.

Yes, go 1.16+ is required.